Atomic/client.py

```diff
@@ -110,3 +112,11 @@ def iter_subs(tree, key=None):
         for i in range(len(tree)):
             tree[i] = iter_subs(tree[i])
     return tree
+
+
+def ids_same(id1, id2):
+    if id1 == id2:
+        return True
+    if no_shaw(id1) == no_shaw(id2):
```

Comment:
In atomic, the image/container ids are exposed without the sha256 pre-pended.  There was confusion about what sha sum was being used in the local filesystem.  We thought it was the image ID but if it is the manifest, this is no longer needed.